gh-md-toc

```diff
@@ -265,10 +272,15 @@ gh_toc_app() {
         shift
     fi
 
+    if [ "$1" = '--nobackup-insert' ]; then
```

Comment:
Let's rename `--nobackup-insert` into `--no-backup`?
`--no-backup` would be make sense only with `--insert` option.

Reply:
Let's decide how this option will be named :)

Here you're checking `--nobackup-insert`, but in docs it called `--no-backup`

Reply:
Help says:

```
...
$app_name [--ins-nb] src [src]  Create TOC without backup, requires <!--ts--> / <!--te--> placeholders
...
```

But we're checking `--nobackup-insert` option here.